cordova-plugin-firebase.opam

```diff
@@ -1,4 +1,5 @@
 opam-version: "2.0"
+version: "2.0"
```

Comment:
```suggestion
version: "1.0"
```

No?

Reply:
Since it was a broking change (the name of function change) I thought adding a 2.0 would be fitting, but if you think 1.0 is better I will do it on the bs PR too.

Reply:
You are right, it is a breaking change but there was not version number yet, this is why I have suggested to start with version 1.0.